README.md

```diff
@@ -7,12 +7,26 @@ You need someone like Jules Winnfield to save you from the vagaries of SAML logi
 ## Source code visualizer
 [CodeSee codebase visualizer](https://app.codesee.io/maps/public/53e91640-23b5-11ec-a724-79d7dd589517)
 
-# npm library
-Jackson is available as an npm library (https://www.npmjs.com/package/@boxyhq/saml-jackson) that can be integrated into express.js routes. In theory, the library should be usable with other node.js web application frameworks but is currently untested. Please file an issue or submit a PR if you encounter any issues.
+## Getting Started
 
-Refer to https://github.com/boxyhq/jackson#configuration for the configuration options to pass to the library.
+### Install
```

Comment:
@devkiran There are two ways to use this repo - 1) as an npm library or 2) as a separate service. We'll want to keep the instructions separate for the two ways since any user would only pick one of these.

Reply:
Noted.